tdrs-frontend/src/components/NavItem/NavItem.jsx

```diff
@@ -17,9 +17,7 @@ function NavItem({ pathname, tabTitle, href }) {
         key={tabTitle}
         // dash-case the tabTitle string (e.g. Data Files => data-files)
         id={tabTitle.replace(/ /g, '-').toLowerCase()}
-        className={`usa-nav__link ${
-          pathname.includes(href) ? 'usa-current' : ''
-        }`}
+        className={`usa-nav__link ${pathname === href ? 'usa-current' : ''}`}
```

Comment:
This fixes a bug that caused navitems to remain active in 404 pages in certain cases.